website/blog/2024-04-22-release-0.74.md

```diff
@@ -349,6 +349,39 @@ Delete any usages of <code>[application:didRegisterUserNotificationSettings:](ht
 
 </details>
 
+### Removal of Flipper React Native Plugin
+
+Use of [Flipper](https://fbflipper.com/) for inspecting React Native layouts, network requests, and [other React Native plugin features](https://www.internalfb.com/intern/staticdocs/flipper/docs/features/react-native/), is now unsupported. In 0.74, we have removed the native Flipper libraries and setup code from new React Native projects. This means fewer dependencies and quicker local setup.
+
+The diff for removing Flipper in your app can be seen in the [Upgrade Helper](https://react-native-community.github.io/upgrade-helper/). If you want to preserve Flipper in an existing app, ignore the relevant diff lines.
+
+- [Original RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0641-decoupling-flipper-from-react-native-core.md)
+
+<details>
+<summary>**💡 To re-integrate Flipper (unsupported)**</summary>
+
+Flipper can still be used as a standalone tool for debugging an Android or iOS app, and can be manually integrated by following the Flipper docs ([Android guide](https://fbflipper.com/docs/getting-started/android-native/), [iOS guide](https://fbflipper.com/docs/getting-started/ios-native/)).
+
+We recommend that teams invest in switching to native debugging tooling in Android Studio and Xcode.
+
+</details>
+
+:::tip
+#### Replacing Flipper
+
+There are a number of dedicated debugging tools which replace Flipper features. For more information, we recommend reading the excellent [*Why you don't need Flipper in your React Native app*](https://shift.infinite.red/why-you-dont-need-flipper-in-your-react-native-app-and-how-to-get-by-without-it-3af461955109) article by Jamon Holmgren.
+:::
+
+#### JavaScript debugging
```

Comment:
Changes planned. To simplify removing Flipper wholesale in 0.74, we plan to update "Open Debugger" to be "Open Debugging Docs" (to the default **Hermes Debugger** flow), outside `--experimental-debugger`.